app/views/support_interface/uploads/preview.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-grid-row">
```

Comment:
We should probably add page titles to this and the other support interface pages. They should match the `<h1>` on each one.

Reply:
https://github.com/DFE-Digital/check-childrens-barred-list/pull/95/commits/44cb74dcfc77008bbaedae4c85568138f3e68f1e